app/src/main/java/org/p2p/wallet/sdk/facade/SolendSdkFacade.kt

```diff
@@ -0,0 +1,211 @@
+package org.p2p.wallet.sdk.facade
+
+import com.google.gson.Gson
+import org.p2p.solanaj.rpc.NetworkEnvironment
+import org.p2p.wallet.home.model.Token
+import org.p2p.wallet.infrastructure.dispatchers.CoroutineDispatchers
+import org.p2p.wallet.infrastructure.network.environment.NetworkEnvironmentManager
+import org.p2p.wallet.sdk.SolendSdk
+import org.p2p.wallet.sdk.facade.mapper.SolendMethodResultMapper
+import org.p2p.wallet.sdk.facade.model.SolendConfigResponse
+import org.p2p.wallet.sdk.facade.model.SolendEnvironment
+import org.p2p.wallet.sdk.facade.model.SolendFeePayerTokenData
+import org.p2p.wallet.sdk.facade.model.SolendMethodResultSuccess
+import org.p2p.wallet.sdk.facade.model.SolendMethodResultSuccess.SolendCollateralAccount
+import org.p2p.wallet.sdk.facade.model.SolendMethodResultSuccess.SolendCollateralAccountsList
+import org.p2p.wallet.sdk.facade.model.SolendMethodResultSuccess.SolendDepositTransactions
+import org.p2p.wallet.sdk.facade.model.SolendMethodResultSuccess.SolendMarketInformation
+import org.p2p.wallet.sdk.facade.model.SolendMethodResultSuccess.SolendTokenDepositFees
+import org.p2p.wallet.sdk.facade.model.SolendMethodResultSuccess.SolendUserDeposit
+import org.p2p.wallet.sdk.facade.model.SolendMethodResultSuccess.SolendUserDepositByTokenResponse
+import org.p2p.wallet.sdk.facade.model.SolendMethodResultSuccess.SolendUserDepositsResponse
+import org.p2p.wallet.sdk.facade.model.SolendMethodResultSuccess.SolendWithdrawTransactions
+import org.p2p.wallet.utils.Base58String
+import timber.log.Timber
+import kotlinx.coroutines.withContext
+
+private const val TAG = "SolendSdkFacade"
+
+class SolendSdkFacade(
```

Comment:
Let's call it repository please, don't go out to new namings

Reply:
that's the idea!
it's not Repository, but just a Facade that should be used in Solend multiple repositories, we should split Solend different coupled methods in separate classes and not create garbage in one class